features/dd-sdk-android-flags/src/main/kotlin/com/datadog/android/flags/Flags.kt

```diff
@@ -79,7 +90,13 @@ object Flags {
             internalLogger.log(
                 InternalLogger.Level.ERROR,
                 InternalLogger.Target.MAINTAINER,
-                { "Missing required context parameters: $missingParams" }
+                { "Missing required configuration parameters: $missingParams" }
+            )
+
+            internalLogger.log(
+                InternalLogger.Level.WARN,
+                InternalLogger.Target.USER,
+                { "Missing required configuration parameters: $missingParams" }
             )
 
             return null
```

Comment:
Do we prefer null here or a NoOpFlagsClient? Also, after this we could probably remove the TODO on line 82

Reply:
null here is fine; we are dropping this create-on-enable routine anyway and will guarantee a non-null Client from the create/get methods